### ISSUE TITLE
🐛 Greeter doesn't have "Sway (Unsupported GPU)" option mentioned in FAQ

### ISSUE THREAD

**Issue by komali2** [manjaro-sway/manjaro-sway#886]
### branch

stable

### Architecture

x86_64

### Description

In https://github.com/manjaro-sway/manjaro-sway/blob/main/SUPPORT.md#why-doesnt-it-start-with-nvidia-drivers, a greeter option is mentioned that I don't seem to have. I only have sway and sway-user-service. 

I selected "proprietary drivers" during installation. 

### Reproduction steps

```bash
Install and select proprietary drivers 

Observe no listed greeter option for unsupported gpu
```

**Comment by boredland** [comment]
Ah, yes. That is outdated. Calling sway should detect if you're using the Nvidia driver and start sway accordingly: https://github.com/manjaro-sway/desktop-settings/blob/sway/community%2Fsway%2Fusr%2Flocal%2Fbin%2Fsway

Does it work?

**Comment by komali2** [comment]
Ah, thank you!

It doesn't seem to be working for me, I'm having issues wherein either nothing displays after login, or only this text displays indefinitely:

![P_20241214_235801](https://github.com/user-attachments/assets/f0f0d3dd-7ca3-4fe7-aeaf-4e5ba6562dca)


I can get to tty without issue. I tried launching `sway` from there to see if I can get an error to display, but it just shows the same text, except then also locks up and prevents me from switching to another tty. 

Furthermore, during installation, there were significant graphical glitches on the installer, such as "stuck rendering" (scrolling showing outdated information in a list until I highlighted a line and forced a refresh for example) or constant flickering. 

I chalked it up to me not quite understanding things and was going to do more investigation along the lines of running Sway on an Nvidia GPU before I bothered anyone here or elsewhere, knowing that Nvidia GPUs are unsupported. Though of course any pointers I'd gladly and gratefully accept.
